apps/nextjs/src/app/api/chat/chatHandler.ts

```diff
@@ -0,0 +1,255 @@
+import {
```

Comment:
By breaking this out into a separate file we can write tests without needing to deal with Clerk / Middleware etc.